core/src/main/scala/org/http4s/QValue.scala

```diff
@@ -129,7 +129,7 @@ object QValue {
   @deprecated("""use qValue"" string interpolation instead""", "0.20")
   def q(d: Double): QValue = macro Macros.qValueLiteral
 
-  implicit val http4sOrderForQValue: Order[QValue] = Order.fromOrdering[QValue]
+  implicit lazy val http4sOrderForQValue: Order[QValue] = Order.fromOrdering[QValue]
```

Comment:
It's not immediately clear to me why this became necessary.  Without it, we get:

```scala
[error] cannot create an instance for class org.http4s.testing.ResponseGeneratorSpec
[error]   caused by java.lang.ExceptionInInitializerError
[error]   caused by scala.UninitializedFieldError: Uninitialized field: /home/travis/build/http4s/http4s/core/src/main/scala/org/http4s/QValue.scala: 132
```

Reply:
Okay, this is now causing a stack overflow error.  I'm reaching the stupid hours... what's suddenly wrong with this?

Reply:
The issue is the new import pulling in `catsKernelOrderingForOrder`, which has higher precedence than the `Ordering`-from-`Ordered` stuff. You could either hide it, or only import `catsSyntaxEither`, or (what I'd do) not use `valueOr`.

Reply:
Oof, thanks. I'd moved on to a different branch and was stupefied by what import was even causing it. As usual, I should have quit an hour before I did.